subsys/bluetooth/host/audio/media_proxy.h

```diff
@@ -494,24 +573,36 @@ struct media_proxy_ctrl_cbs {
 	 *                 or errno on negative value.
 	 * @param state    The media player state
 	 */
-	void (*media_state)(struct media_player *player, int err, uint8_t state);
+	void (*media_state_recv)(struct media_player *player, int err, uint8_t state);
+
+	/**
+	 * @brief Command send callback
+	 *
+	 * Called when a command has been sent
+	 * See also media_proxy_ctrl_command_send()
+	 *
+	 * @param player   Media player instance pointer
+	 * @param err      Error value. 0 on success, GATT error on positive value
+	 *                 or errno on negative value.
+	 * @param cmd      The command sent
+	 */
+	void (*command_send)(struct media_player *player, int err, struct mpl_cmd cmd);
```

Comment:
I think we should modify the tense of the words here. 

Typically we have used past tense for callbacks, e.g. `connected`, `written`, `sent`, `received`, `updated`, etc. 

I guess this should be changed to `command_sent` and the write callbacks as `x_written`. 

The "read" callbacks can stay `recv` as that could be `received`

Reply:
This is the naming that was agreed upon in the discussion with @joerchan, and which no-one objected to.
The reasons to have "write" and "recv" were that those are the names used in other parts of the stack.

Reply:
I completed missed @joerchan replies in that thread somehow. He is correct that we use `read`/`write` for callbacks in other places, but we also have `_written` in the `ots` API, and general (as per my examples), the general tense is past tense for callbacks. 

But if you and Joakim prefers `write/receive` then we'll just do that.

Reply:
Personally, I am fine with "_written" as well, but since I had asked for his opinion (I actually think you suggested that) I thought I should follow it ;-)

Reply:
My preferrence is to use past-tense for callbacks, but it's fine as it then :) It's still an experimental API and can change later if we regret it.